adapters/bluesea/bluesea.go

```diff
@@ -67,6 +67,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 			Uri:     fmt.Sprintf("%s?%s", a.endpoint, queryString),
 			Body:    reqJson,
 			Headers: headers,
+			ImpIDs:  []string{imp.ID},
```

Comment:
This is an odd one. The code looks like it's splitting the imps to their own request, but the request still includes all of the imps. So, this should use GetImpIDs on the request. Might be a design flaw with the adapter, but let's focus on the code.

Reply:
thanks for noticing it.
Done